pkg/manifests/types.go

```diff
@@ -258,6 +258,9 @@ type ThanosQuerierConfig struct {
 	// The possible values are `error`, `warn`, `info`, and `debug`.
 	// The default value is `info`.
 	LogLevel string `json:"logLevel,omitempty"`
+	// A Boolean flag that enables setting CORS headers.
+	// The default value is `false`.
```

Comment:
```suggestion
	// The default value is `false`.
	// In enabled, Thanos sets CORS headers to be allowed by all.
```

Reply:
I assume you're talking about `Access-Control-Allow-Origin: *`

How about

```
// A Boolean flag that enables setting CORS headers.
// The headers would allow access from any origin.
// The default value is `false`.
```

Reply:
I pushed a this version.